src/others/helpers/map-utils.ts

```diff
@@ -1,160 +1,51 @@
-import { groupBy } from "lodash";
-import omit from "lodash/omit";
 import { AidRequest, Supply, Location } from "../contexts/api";
-import { DecodedAidRequest, DecodedAidRequestGroupedByLocation } from "./decode-aid-request";
 import type { Feature, Geometry, GeoJsonProperties } from "geojson";
 
-type GroupedByDate = {
-  date: string;
-  aidRequests: Omit<DecodedAidRequest, "date">[]; // TODO: consider replacing with AidRequest
-};
-
-type GroupedByDateWithTotal = {
-  date: string;
-  aidRequests: Omit<DecodedAidRequest, "date">[]; // TODO: consider replacing with AidRequest
+type GroupedByCityId = {
+  city_id: number;
   total: number;
-  totalDescription: string;
-};
-
-type GroupedByLocationAndDateWithTotal = {
-  location: Location;
-  aidRequestsByDateWithTotal: GroupedByDateWithTotal[];
+  aidRequests: AidRequest[];
 };
 
-interface RequestProperties {
+type MapData = {
```

Comment:
**suggestion**
Maybe `RequestMapDataPoint`. MapData is very vague and can mean anything.